pom.xml

```diff
@@ -175,6 +181,11 @@
 			<artifactId>netty-common</artifactId>
 			<version>${netty.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>io.netty</groupId>
+			<artifactId>netty-buffer</artifactId>
+			<version>${netty.version}</version>
+		</dependency>
```

Comment:
You seem to add this dependency but do not really use it. Is it needed as some GSON override? Or why?

Reply:
I still don't follow why do you have this change here.

Reply:
Tbh I am confused as well Jakub.
Without adding this dependency I am getting:

```shell
[WARNING] Used undeclared dependencies found:
[WARNING]    io.netty:netty-buffer:jar:4.1.85.Final:compile
```

But I am not sure where, in the new code, I am using something from this JAR :-(
It was the same with GSON and now again with Jackson.

Reply:
Ok, mysterious way of Maven plugins ... but not much we can do about it other than add it. Thanks for the explanation.

Reply:
Let's say that the next step will be removing the Vert.x `Buffer` class which is anyway using Buffer from Netty, and I will use it directly, so the jar will be needed at that point :-)